Analysis/Physics/Chic2invi/Chic02ee/Chic02ee-00-00-01/src/Chic02ee.cc

```diff
@@ -458,17 +425,7 @@ evtflw_filled = true;
 } 
 return nlplm; 
```

Comment:
fix this bug:
not correct to assign 1 to nlplm in line 403.

Reply:
I have assigned at line **308:** to the selectLeptonPlusLeptonMinus(evtRecTrkCol, iPGood, iMGood0 !=1) return false;
If int nlplmwill be other than 1, it will not work.

Reply:
try to answer the following questions:

-  what is the meaning of nlplm?
-  explain the meaning of selectLeptonPlusLeptonMinus(evtRecTrkCol, iPGood, iMGood0 !=1) return false;   why do you need this requirement?